src/web/src/components/ScheduleEvent.vue

```diff
@@ -1,7 +1,7 @@
 <template functional>
   <div class="schedule-event" :style="data.style" data-cy="schedule-event">
     <div class="event-text">
-      <span data-cy="title">{{ props.title }}</span>
+      <span data-cy="title"> <a v-bind:href= "'/explore/' + props.name.split(' ')[0] + '/' + props.name.split(' ')[0] + '-' + props.name.split(' ')[1]"> {{props.title}}</a> </span>
```

Comment:
What do you think about moving this string/url formatting into a separate function?

@123joshuawu refactored something similar here: https://github.com/YACS-RCOS/yacs.n/pull/319/files

For names maybe we can do func/property `getExploreCourseLink`/`exploreCourseLink`.

wyt?

Reply:
Yeah that sounds great! I wasn't sure how to do it that way but I'll check how Josh did it